panic: runtime error: index out of range

Hi!
If in CLI I press enter without any command the program panic.

I'm using last docker image
```
config> info
Debug mode (server): false
Hostname: 6e60ad0b0a2f
CPUs: 4
Demo mode: false
DB Driver: sqlite3
DB Conn: ./sshportal.db
Bind Address: :2222
System Time: 2017-12-15T08:45:50.935254557Z
OS Type: linux
OS Architecture: amd64
Go routines: 9
Go version (build): go1.9.2
Uptime: 7.928332912s
User email: 1
User email: admin@sshportal
Version: 1.6.0+dev
GIT SHA: a413aa86c277941b831ecbe3dcecaec117d5a0b5
GIT Branch: master
GIT Tag: a413aa8

config> version
1.6.0+dev
```


```
2017/12/15 08:44:23 New connection: sshUser="admin" remote="172.16.1.36:22407" local="172.17.0.2:2222" command=[] dbUser=id:'\x01',email:admin@sshportal
panic: runtime error: index out of range

goroutine 81 [running]:
main.(*Event).SetArg(...)
        /go/src/github.com/moul/sshportal/shell.go:1921
main.shell(0xc42027a420, 0xdf80a0, 0xc42031f6b0, 0x0, 0x0, 0x0, 0xc4200f62d0, 0x0, 0x0)
        /go/src/github.com/moul/sshportal/shell.go:1921 +0x884c
main.server.func2(0xdf80a0, 0xc42031f6b0)
        /go/src/github.com/moul/sshportal/main.go:135 +0x17ef
github.com/moul/sshportal/vendor/github.com/gliderlabs/ssh.(*session).handleRequests.func1(0xc42031f6b0)
        /go/src/github.com/moul/sshportal/vendor/github.com/gliderlabs/ssh/session.go:231 +0x3b
created by github.com/moul/sshportal/vendor/github.com/gliderlabs/ssh.(*session).handleRequests
        /go/src/github.com/moul/sshportal/vendor/github.com/gliderlabs/ssh/session.go:230 +0x4e2
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,7 +2,7 @@
 
 ## master (unreleased)
 
-* No entry
+* Fix panic when entering empty command (fix [#13](https://github.com/moul/sshportal/issues/13))
 
 ## v1.6.0 (2017-12-12)
 

--- a/shell.go
+++ b/shell.go
@@ -1918,6 +1918,9 @@ GLOBAL OPTIONS:
 			if len(words) == 1 && strings.ToLower(words[0]) == "exit" {
 				return s.Exit(0)
 			}
+			if len(words) == 0 {
+				continue
+			}
 			NewEvent("shell", words[0]).SetAuthor(&myself).SetArg("interactive", true).SetArg("args", words[1:]).Log(db)
 			if err := app.Run(append([]string{"config"}, words...)); err != nil {
 				if cliErr, ok := err.(*cli.ExitError); ok {

